statistics-production/ees.qmd

```diff
@@ -433,87 +450,88 @@ This then becomes the default order for this filter or indicator and will apply
+* Infographic pages
+* Supplementary data that isn't intended for the table tool
 
-![](../images/replace-data-start.png)
+To ensure that spreadsheets are accessible, see the guidance from [gov.wales](https://gov.wales/how-create-accessible-excel-spreadsheets), and this [Analysis Function guidance](https://analysisfunction.civilservice.gov.uk/policy-store/making-analytical-publications-accessible/).
 
-The first step is to upload the new file.
+If you are unsure of whether you should be using supporting files, contact the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk) for advice.
 
-![](../images/replace-data-step1.PNG)
+---
 
-One you've chosen and uploaded your replacement file it will need to go through the usual import process before it can check if retaining existing data blocks and footnotes will be possible.
+### Public data guidance
 
-![](../images/replace-data-step2.PNG)
+---
 
-Once the upload is finished a report will appear which highlights whether existing data blocks and footnotes can be replaced successfully. If you want to keep any data blocks and footnotes you've built you will need to make sure that your replacement data file still contains the information (indicators, filters, geographic_levels and time_periods) that was used to create them.
+Within the 'Data and files' page, you can also create your public data guidance. This replaces the information that would have previously been uploaded as a pdf on GOV.UK and is designed to help users understand and use the data they download from your release. See the [Permanent and fixed period exclusions data guidance](https://explore-education-statistics.service.gov.uk/find-statistics/permanent-and-fixed-period-exclusions-in-england/meta-guidance) for an example.
 
-![](../images/replace-data-success.PNG)
+The 'Data files' section of the document will automatically update as you add new data files to your release, however you will need to add an overview of all the data included in the release and short summaries for each data file before the release can be published.
 
-If it's not possible for a data block or footnote to be recreated using the replacing data file a warning will appear and you'll be prompted to either edit or delete them before completing the replacement.
+![](../images/admin-public-meta-start.PNG)
 
-![](../images/replace-data-fail.PNG)
+A list of variables in each file with an associated label (taken from metadata uploads) and associated footnotes will also be displayed for each file.
+
+![](../images/admin-public-meta-detail.PNG)
 
 <div class="alert alert-dismissible alert-danger">
-Remember to double check any data blocks or footnotes that were recreated by the platform before publishing your release.
+Releases cannot be published without a completed data guidance. If not filled in an error will be flagged during sign off.
```

Comment:
"a completed data guidance" sounds odd - is there a word missing?

Reply:
Good spot, added